code/game/gamemodes/changeling/implements/powers/hivemind.dm

```diff
@@ -0,0 +1,19 @@
+//To get rid of pesky moaners
+/mob/proc/changeling_eject_hivemind()
+	set category = "Changeling"
+	set name = "Hivemind Eject"
+	set desc = "Ejects a hivemind from ourselves."
+
+	var/list/mob/hiveminds_to_eject = list()
+
+	for(var/mob/abstract/hivemind/H in src.mind.changeling.hivemind)
```

Comment:
You don't need to do this. You can just make the `input()` on line 13 into:

```suggestion
	input(...) as anything|null in mind.changeling.hivemind
```

And adjust the following code.